beacon.go

```diff
@@ -30,25 +31,40 @@ import (
 )
 
 type beaconConfig struct {
-	projectID string
-	tableId   string
+	ApiVersion string
+	ProjectID  string
+	TableId    string
 }
 
 const (
-	projectKey = "GOOGLE_CLOUD_PROJECT"
-	bqTableKey = "GOOGLE_BIGQUERY_TABLE"
+	apiVersionKey = "VERSION"
+	projectKey    = "GOOGLE_CLOUD_PROJECT"
+	bqTableKey    = "GOOGLE_BIGQUERY_TABLE"
 )
 
-var config = beaconConfig{
-	projectID: os.Getenv(projectKey),
-	tableId:   os.Getenv(bqTableKey),
-}
+var (
+	aboutTemplate = template.Must(template.ParseFiles("about.xml"))
+	config        = beaconConfig{
+		ApiVersion: os.Getenv(apiVersionKey),
+		ProjectID:  os.Getenv(projectKey),
+		TableId:    os.Getenv(bqTableKey),
```

Comment:
ID or Id?  A format should be consistent across the codebase.  Just pick one and stick with it.

Reply:
Not all of them are really keys, but I know what you are intending here.  For now it's okay, but try to keep the names to what they mean.  For example there are API keys and some might interpret `apiVersionKey` as a key for a specific version of the API, which could be confusing with future versions of this codebase.